docs_src/tutorials/kubernetes.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes
+
+This tutorial uses [Helm](https://helm.sh/docs/intro/quickstart/) to install
+Scaphandre, Prometheus and Grafana.
+
+## Install Scaphandre
+
+First we install Scaphandre which runs as a daemon set which creates a pod on
+each node for collecting the metrics.
+
+    helm install scaphandre helm/scaphandre
+
+## Install Prometheus
+
+Next we will install Prometheus which will scrape the metrics generated by Scaphandre.
+
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
+    helm repo update
+
+    helm install prometheus prometheus-community/prometheus \
+    --set alertmanager.persistentVolume.enabled=false \
+    --set server.persistentVolume.enabled=false
+
+This setup should only be used for testing as the Prometheus data is not
+persisted if the pods are deleted.
+
+You can access the Prometheus web UI by creating a port forwarding connection.
+
+    kubectl port-forward deploy/prometheus-server 9090:9090
+
+## Install Grafana
+
+Create a configmap to store the Grafana dashboard.
+
+    kubectl create configmap scaphandre-dashboard \
+        --from-file=scaphandre-dashboard.json=docs_src/tutorials/grafana-kubernetes-dashboard.json
```

Comment:
This was the best way I could find to install the dashboard. Happy to move the files if they would be better somewhere else.

Reply:
Maybe we could find a better location later. I guess this is fine for now.

Reply:
Yes I actually wanted to try loading the dashboard from Grafana Cloud here
https://grafana.com/grafana/dashboards/13845

But I ran into a bug with the Grafana chart.
https://github.com/grafana/helm-charts/issues/200